leits/MeetingBar
# Remove html tags from meeting description

### Status
**READY**

### Description
When you use the event description's formatting in Google Meet, it adds HTML tags to the events description/notes, which so far just shows the plain HTML code in MeetingBar. This PR strips those tags.

#### Before
<img width="1016" alt="Screen Shot 2021-01-11 at 22 22 48" src="https://user-images.githubusercontent.com/15385891/104240453-613c5380-545c-11eb-9f7a-55f1c034e1b8.png">

#### After
<img width="276" alt="Screen Shot 2021-01-11 at 22 23 20" src="https://user-images.githubusercontent.com/15385891/104240471-67cacb00-545c-11eb-9592-09a8ee11616b.png">

### Steps to Test or Reproduce
1. Create a meeting in Google Meet.
2. Add a description with a some formatting (e.g. bullet point list, like in the screenshot).
3. Check the meetings' details view in MeetingBar.

### Note
This PR also removes the hardcoded dev team from the project file. This is required to make the `XCConfig/DevTeamOverride.xcconfig` work (great idea btw!), because otherwise the project file always wins. I hope this doesn't break anything for you or on CI.
